### ISSUE TITLE
GTEx TMT sample mapping

### ISSUE THREAD

**Issue by helenabea** [bzhanglab/PepQuery#59]
Hello, I'm having a lot of difficulty understanding how to map pepquery output to TMT tag of GTEx so that I can correctly map peptide to sample_ID.

I know this issue was opened before, but I do not understand the answers, mainly because this is my first time using mass-spec data.

So basically, I have this peptide:

spectrum_title: SecondInstrument_Sample20_S2R23_061716_Fr04:25026:4  
peptide: HQDWRSGEAEFPDEESDESSSTSETSEEEVQDR          
modification: TMT_10-plex_of_peptide_N-term@0[229.1629]
exp_mass: 4042.6508257733763  
pep_mass: 4042.68993351836    
tol_ppm: 9.67369390842789      
tol_da: 0.039107744983539305    
isotope_error: 0.0            
score: 17.858883971787446

I was able to track it to psm_rank.mgf file, which gives me the following information:

BEGIN IONS
TITLE=SecondInstrument_Sample20_S2R23_061716_Fr04:25026:4
PEPMASS=1011.669982910156 3234025.828125
RTINSECONDS=8040.620801
CHARGE=4+

However, I do not understand how I can map this to GTEx sample ID. 
![image](https://github.com/bzhanglab/PepQuery/assets/48881948/62cc364f-e0ac-49a3-bcd7-077c2245566d)
So basically, I do understand that SecondInstrument_Sample20 has samples from Esophagus - Muscularis, Nerve - Tibial, Spleen, Uterus, Liver, Heart - Left Ventricle, Skin - Sun Exposed (Lower leg). What I want is to correctly attribute peptide "HQDWRSGEAEFPDEESDESSSTSETSEEEVQDR" to the "Nerve - Tibial" sample, for example.

**Comment by wenbostar** [comment]
The GTEx dataset is generated using the SPS-MS3 workflow as described in this paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7575058/. In this workflow, peptide identification is based on MS2 spectra while quantification is based on MS3 spectra. You won't be able to get the quantification information from the spectra returned by PepQuery since the identification is based on MS2 spectra which don't contain quantification data. I'd like to suggest to use [FragPipe](https://github.com/Nesvilab/FragPipe) for quantification. You could add the target peptides identified by PepQuery into a database then use FragPipe to quantify these peptides.
